debian/rules

```diff
@@ -9,6 +9,8 @@ export DH_GOLANG_EXCLUDES=tests
 export DH_GOLANG_GO_GENERATE=1
 
 export PATH:=${PATH}:${CURDIR}
+# make sure that correct go version is found on trusty
```

Comment:
We couldn't update golang-go in trusty-updates without causing a whole heap of FTBFSs but we can add new packages, and that's what we did. This is the correct thing to do.